### PR TITLE
updated the dark mode and added only the required file

### PR DESCRIPTION
**Pull request by Tejas1510**

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Can you attach a screenshot of the changes? In the preview, it is difficult to check the dark mode without a toggle button

**Comment by Tejas1510** [comment]
Sir i have added a toggle button in the code which i have pushed 
For your simplicity you can check the experience page here:
https://gracious-benz-a98fca.netlify.app/experience.html 

There is a toggle button which will toggle between dark and light mode

Also i am attaching the following screenshot

Screenshot (72)

Screenshot (71)

File: awesome-portfolio-websites/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/CODE_OF_CONDUCT.md:7]: Do not alter this please

File: awesome-portfolio-websites/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<h1 align="center"><b> How to Contribute? </b></h3> 
+
+
+We value all the people who are interested in contributing to Girlscript website boilerplate. This is required to create a unified development environment.These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/CONTRIBUTING.md:4]: Do not alter this please

File: awesome-portfolio-websites/LICENSE [hunk 3]
```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
+
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/LICENSE:2]: Do not alter this please

File: awesome-portfolio-websites/README.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
+<h1 align= "center"><b>Awesome Portfolio Websites</b></h1>
+An open source project for creating end-end templates for portfolio websites for everyone to use. We are a clan of geeks who wants to provide you with that dope website that flaunts your hardwork and projects over the years to your mates and employers! With a deep desire to have that satiating UI, those engaging animations,the great SEO and the optimized code, we do it all. Hop in and give your contributions if you wish to be a part of this initiative.
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/README.md:2]: Do not alter this please

File: awesome-portfolio-websites/robots.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+User-agent: Googlebot
+User-agent: AdsBot-Google
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/robots.txt:2]: Do not change this please

File: awesome-portfolio-websites/research.html [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+        </table>
+      </div>
+    </div>
+  </div>
+
+  <!-- Dynamic footer section -->
+
+  <!-- Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-167979537-2"></script>
+  <!-- Fetching our Google Tag Manager -->
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/research.html:193]: Do not change this please

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Your commit seems to alter a lot of files that is not required for this issue.
Please only stage the changes you were assigned

**Comment by smaranjitghose** [comment]
toggle button
The  toggle button is poorly placed and barely visible.
- Due to a recent PR, we have a toggle button as you can see [here](https://portfolio.smaranjitghose.codes/). Please make this consistent for your PR as well

**Comment by smaranjitghose** [comment]
illustration
Please consult @anushbhatia for the Dark Mode UI and color scheme. The current one doesn't seem aesthetically appealing
erroff
These are some bugs to be fixed

Please do the changes ( and stage + commit only the files you are concerned with) and give a new PR

**Pull request closed without merging**